skeleton-typescript-webpack/package.json

```diff
@@ -113,6 +113,7 @@
     "@types/webpack": "^2.0.0",
     "@types/whatwg-fetch": "^0.0.32",
     "aurelia-protractor-plugin": "^1.0.1",
+    "babel-polyfill": "^6.16.0",
```

Comment:
The "wallaby-webpack" module uses Promise. Without the babel-polyfill dependency it gives me this error:
```
1 failing tests, 9 passing

test/unit/app.spec.ts the App module contains a router property [5 ms]

	ReferenceError: Can't find variable: Promise
		at src/users.ts:5

	ReferenceError: Can't find variable: Promise
	at http://localhost:65114/wallaby-webpack.js?1480348701747:1
```

I can try to also add it to the esnext skeleton.